### PR TITLE
790 yc outoutassortativity and inoutassortativity

### PR DESCRIPTION
**Pull request by c-yuwei**

### CODE REVIEW

File: braph2genesis/measures/_InOutAssortativity.gen.m [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+%%% ¡compatible_graphs!
+GraphBD
+GraphWD
+MultiplexBD
+MultiplexWD
+
+%% ¡props_update!
+
+%%% ¡prop!
+M (result, cell) is the richness.
```
Comment (annacanal) [inline braph2genesis/measures/_InOutAssortativity.gen.m:28]: in-out-assortativity, not richness

File: braph2genesis/measures/_OutOutAssortativity.gen.m [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+%%% ¡compatible_graphs!
+GraphBD
+GraphWD
+MultiplexBD
+MultiplexWD
+
+%% ¡props_update!
+
+%%% ¡prop!
+M (result, cell) is the richness.
```
Comment (annacanal) [inline braph2genesis/measures/_OutOutAssortativity.gen.m:28]: out-out-assortativity. not richness

**Review by annacanal — CHANGES_REQUESTED** [review]

File: braph2genesis/measures/_InOutAssortativity.gen.m (hunk 1 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InOutAssortativity.gen.m]: 15d643b5

File: braph2genesis/measures/_OutOutAssortativity.gen.m (hunk 2 above)
Comment (c-yuwei) [reply braph2genesis/measures/_OutOutAssortativity.gen.m]: 15d643b5

**Review by annacanal — APPROVED** [review]

**Pull request merged**